ganalytics.php

```diff
@@ -279,13 +291,27 @@ public function onAssetsInitialized()
             $code .= $this->getOptOutCode($trackingId, $optout_config);
         }
         $settings = $this->getTrackingSettings($trackingId, $objectName);
-        $code .= $this->getTrackingCode($scriptName, $objectName, $async);
+
+        if ($isNewMode) {
+          $code .= $this->getGa4TrackingCode($objectName);
+        } else {
+          $code .= $this->getOldTrackingCode($scriptName, $objectName, $async);
+        }
+        
+
         $code.= join(PHP_EOL, $settings);
 
         // Embed Google Analytics script
         $group = ($position == 'body') ? 'bottom' : null;
 
         $this->grav['assets']->addInlineJs($code, null, $group);
-        if ($async) $this->grav['assets']->addJs("//www.google-analytics.com/{$scriptName}.js", 9 , true /*pipeline*/, 'async', $group);
+
+        if ($isNewMode) {
+          $this->grav['assets']->addJs("https://www.googletagmanager.com/gtag/js?id={$trackingId}", 9 , true /*pipeline*/, 'async', $group);
+        } else if ($async) {
```

Comment:
**Question:** `async` setting is removed in blueprint. How this will work if I save settings in Admin with UA tag?